changelogs/fragments/830-pam-limits.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - pam_limits - adds check mode (https://github.com/ansible-collections/community.general/issues/827)
+  - pam_limits - adds diff mode (https://github.com/ansible-collections/community.general/issues/828)
```

```suggestion
  - pam_limits - adds check mode (https://github.com/ansible-collections/community.general/issues/827).
  - pam_limits - adds diff mode (https://github.com/ansible-collections/community.general/issues/828).
```
according to the doc

---

done